tests/test_hssm.py

```diff
@@ -267,3 +267,14 @@ def test_override_default_link(caplog, data_ddm_reg):
 
     assert "t" in caplog.records[0].message
     assert "strange" in caplog.records[0].message
+
+
+def test_resampling(data_ddm):
```

Comment:
we should put a not somewhere concerning the resulting behavior.
If I understand correctly, if you change sample_2 you automatically change model.traces?

Reply:
I updated the `return` section of the `sample()` method and `traces` object to indicate that `.sample()` returns a reference to `model.traces` object, which stores the traces of the last call to `.sample()`.